extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1783,7 +1788,11 @@ export class AmpStoryPage extends AMP.BaseElement {
 
       this.mutateElement(() => {
         textEl.textContent = openLabel;
-        this.element.appendChild(this.openAttachmentEl_);
+        const container = this.element.ownerDocument
+          .createElement('div')
+          .classList.add('shadow-root-container-element');
```

Comment:
This might be breaking since `.createElement` doesn't return the node.
You might need to do:
`container.classList.add(...`